internal/shared/disk/types.go

```diff
@@ -10,3 +10,8 @@ type DiskLocation struct {
 	Target  string
 	LUNID   string
 }
+
+// DiskID definition
+type DiskID struct {
+	Page83 string
```

Comment:
See above note about potentially making this more extensible through a map of <ID-type, ID> rather than hardcoding the ID types.